### DISCUSSION TITLE
JS for both Breeze and Luma

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#283] [5. General]
Hi @vovayatsyuk 

We are developing a theme based on Breeze and want to write some JS which should work with or without Breeze as that has some common JS which runs on all pages with checkout also and also we can exclude some pages from Breeze so in that case JS should work.

I am writing JS with this way

```
define(["jquery"], function ($) {
    "use strict";

    // top panel close button
    $(".top-message-bar .pagebuilder-button-link").on("click", function () {
        const _this = $(this);
        _this.closest(".top-message-bar").addClass("hidden");
    });

});
 ```
Is this the right way and would not impact site performance ?

**Comment by vovayatsyuk** [comment]
Yes, that's the approach we use in swissup modules.
